Sources/DatadogSDKTesting/DDTestMonitor.swift

```diff
@@ -263,15 +251,9 @@ internal class DDTestMonitor {
 
         itrWorkQueue.addOperation { [self] in
             // Activate Coverage
-            if DDTestMonitor.env.coverageEnabled ?? itrBackendConfig?.codeCoverage ?? false {
+            if DDTestMonitor.config.coverageEnabled && itrBackendConfig?.codeCoverage ?? false {
                 Log.debug("Coverage Enabled")
-                // Coverage is not supported for swift < 5.3 (binary target dependency)
-                #if swift(>=5.3)
-                    coverageHelper = DDCoverageHelper()
-                #else
-                    coverageHelper = nil
-                #endif
-
+                coverageHelper = DDCoverageHelper()
             } else {
```

Comment:
it's no longer the case?

Reply:
I dropped support of swift 5.3. It was broken for a long time already